tests/suite/system_suite_test.go

```diff
@@ -173,6 +174,11 @@ func setup(cfg setupConfig, extraInstallArgs ...string) {
 		}
 		installCfg.ImageTag = *imageTag
 		installCfg.ImagePullPolicy = *imagePullPolicy
+	} else {
+		if version == "edge" {
+			chartVersion = "0.0.0-edge"
+			installCfg.ChartVersion = chartVersion
```

Comment:
The `chartVersion` variable just seems unnecessary. In cases where the version should just be an empty string, let's just not specify `ChartVersion` at all, since the default is an empty string.

Reply:
`chartVersion` is used in the upgrade test, after the setup. I was following the same logic of the other variables like `localChartPath`.

Reply:
Ok I see